include/aspect/parameters.h

```diff
@@ -166,6 +166,7 @@ namespace aspect
     double                         start_time;
     double                         CFL_number;
     double                         maximum_time_step;
+    bool                           use_smoothing;
```

Comment:
Could you call this variable `use_entropy_viscosity_smoothing` to make its intend clearer?